packages/builder/src/utils/utils.ts

```diff
@@ -72,13 +106,43 @@ export const getProviderByChainId = (chainId: ChainId) => {
   );
 
   if (!chainConfig) {
-    throw new Error(`chainConfig not found for chain ID ${chainId}`);
+    console.log(`chainConfig not found for chain ID ${chainId}`);
+    return undefined;
```

Comment:
is there a good reason to remove the error here? loading a chain that we don't support sounds like an error

Reply:
Currently we're fetching data form goerli, but goerli is not supported anymore. therefore the error will be thrown all the time.  So just because one chain is not configured properly, it shouldn't break the app.